blocks/single-column-layout-block/layouts/single-column-regular/default.jsx

```diff
@@ -1,21 +1,33 @@
 import React from "react";
 import PropTypes from "@arc-fusion/prop-types";
+import { Stack } from "@wpmedia/arc-themes-components";
 
 const LAYOUT_CLASS_NAME = "b-single-column-regular";
 
 const SingleColumnRegular = ({ children }) => {
 	const [navigation, main, footer] = React.Children.toArray(children);
 
+	if (!navigation && !main && !footer) {
+		return null;
+	}
 	return (
-		<>
-			{navigation ? <header className={`${LAYOUT_CLASS_NAME}-header`}>{navigation}</header> : null}
+		<Stack className={LAYOUT_CLASS_NAME}>
+			{navigation ? (
+				<Stack role="banner" className={`${LAYOUT_CLASS_NAME}__header`}>
+					{navigation}
```

Comment:
We just talked about this in the other ticket, but we should probably make these match as well.

Reply:
updated https://github.com/WPMedia/arc-themes-blocks/pull/1334/commits/2fbb40d57f73f891ae4946b61e189a87ac530b14